routes/web.php

```diff
@@ -87,4 +96,25 @@
     return Inertia::render('PagesAssistants/ModulePraktikum.jsx');
```

Comment:
```suggestion
    return Inertia::render('PagesAssistants/ModulePraktikum');
```
you need to decide, do you want to add the `.jsx` or not? make sure to uniformed things so its easier to read and wont create any ambiguity

Reply:
thank you for the clarification. I’ll make sure to keep the file extensions consistent without using .jsx